app/views/IndexView.scala.html

```diff
@@ -236,8 +249,29 @@ <h2 class="govuk-heading-m">@messages("index.details")</h2>
                 </p>
             }
 
-            @(maybeExcludedTrader, exclusionsEnabled) match {
-                case (Some(_), true) => {}
+            @(maybeExcludedTrader, exclusionsEnabled, hasRequestedToLeave) match {
+                case (Some(excludedTrader), true, false) => {
+                @if(hasSubmittedFinalReturn){
+                    @if(excludedTrader.exclusionReason == 4) {
```

Comment:
if we could use the enum here as well, might be better. I'd be open for you to change the backend to provide it as an enum as well, over an int